packages/pn-commons/src/components/EmptyState.tsx

```diff
@@ -11,6 +11,8 @@ type Props = {
   emptyActionLabel?: string;
   /** Disable sad emoticon */
   disableSentimentDissatisfied?: boolean;
+  /** Enable happy emoticon */
+  enableSentimentSatisfied?: boolean;
```

Comment:
Maybe it would better if it is more generic. Instead of using a boolean, i would pass the icon directly

Reply:
In my opinion, this depends on who should decide about the icon to show, either `EmptyState` or the component which calls it. 
If I (a component) want to use `EmptyState` with a happy face, what should I indicate, "show a happy face" or else "show this icon"? The former enhances uniformity (as the decision is on `EmptyState`, all happy faces will look the same), the latter enhances flexibility.
In fact, a similar argument could be made about the sad face.
I propose to replace both `disableSentimentDissatisfied` and `enableSentimentSatisfied` by a single `sentimentIcon` atrribute, whose type would be 'KnownSentiments | IconType`, where `KnownSentiments` would be an enum with two elements, DISSATISFIED and SATISFIED'. The default would be `KnownSentiments.DISSATISFIED`. What do you think?

Reply:
Yes your solution is good! We can avoid the use of IconType and leave KnownSentiments to maintain (as you said) the uniformity

Reply:
Refactor of `EmptyState` implemented, hope you like it!